src/messenger-shortcuts.js

```diff
@@ -78,6 +92,16 @@ function last(arr) {
   return arr.length === 0 ? undefined : arr[arr.length - 1];
 }
 
+function getCode(index) {
+  return bind[index].charCodeAt(0);
+}
+
+function dismiss() {
+	if(menuOpened) {
```

Comment:
I see where you're going with it, but I think it's a little more complicated. Try this: Open the help dialog, click anywhere on the help dialog (without closing it), and then press Alt+Shift+/ to open the help dialog again.